web/src/components/NavigationBarScrollContainer.tsx

```diff
@@ -59,8 +60,26 @@ const getScrollableWidth = (scrollContainer: RefObject<HTMLDivElement>): number
   return 0
 }
 
-const NavigationBarScrollContainer = ({ children, direction, scrollContainerRef }: PropsType): ReactElement => {
-  const [scrollPosition, setScrollPosition] = useState(0)
+const NavigationBarScrollContainer = ({
+  children,
+  direction,
+  scrollContainerRef,
+  activeIndex,
+}: PropsType): ReactElement => {
+  const getInitScrollPosition = useCallback((): number => {
+    const navBar = document.getElementById('navbar')
```

Comment:
Personal opinion: I don't like these kind of abbreviations. Instead, I would prefer using the complete name, especially if the component is named differently.
```suggestion
    const navigationBar = document.getElementById('navbar')
```